app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,65 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Actor
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.widget.Placeholder
+import coil.compose.AsyncImage
+import coil.compose.rememberImagePainter
+
+@Composable
+fun ActorCard(
+    actorCardViewState: Actor,
+    modifier: Modifier = Modifier,
+) {
+    Card(modifier = Modifier
+        .width(140.dp)
+        .height(200.dp)
```

Comment:
Composable components should be as generic as possible, so they can be reused whenever needed. For this reason the size should not be defined here. Instead it should be passed in the function using a modifier